README.md

```diff
@@ -1,5 +1,11 @@
 # Jupyter Accessibility
 
+## Accessibility Docs
+
+Looking for [accessibility docs?](https://jupyter-accessibility.readthedocs.io/en/latest/accessibility-docs)
```

Comment:
I'm not 100% sure about this formatting or wording but I do like that it is short and to the point